kong/router/utils.lua

```diff
@@ -1,15 +1,14 @@
 local constants = require("kong.constants")
 local hostname_type = require("kong.tools.utils").hostname_type
 local normalize = require("kong.tools.uri").normalize
-local yield = require("kong.tools.yield").yield
 
 
 local type  = type
```

Comment:
Perhaps remove extra space here too, while doing it.

Reply:
I add some extra spaces to align these localizations, see below.

Reply:
@chronolaw, I think we in general stopped doing alignment. Thus I was asking to remove it as it didn't have that alignment anymore anywhere else.

Reply:
done.